src/main/java/com/es/phoneshop/web/ProductListPageServlet.java

```diff
@@ -13,6 +16,15 @@
 import java.util.List;
 
 public class ProductListPageServlet extends HttpServlet {
+
+    private ProductDao productDao;
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        productDao = ArrayListProductDao.getInstance();
```

Comment:
It's supposed to save sample products to the database somewhere in here, and then use dao.findProducts() method to attach them to the request